san/query.py

```diff
@@ -16,5 +19,10 @@ def parse_dataset(dataset):
     return [left, right]
 
 
+@add_transform
+def transform_query(idx, query, result):
```

Comment:
Now that I think this will better fit the `transform.py` module and probably with a different name - `transform_query_result` or just `transform` as it's not the query that's transformed, but the result :?

Reply:
I renamed the function and moved it into transform.py